src/Client/components/ques/written-ques.tsx

```
@@ -1,143 +1,200 @@
+                  </label>
+                )}
+                {!question.isLongAnswer && (
+                  <label>
+                    <Checkbox
+                      checked={question.isLongAnswer}
+                      onChange={(e) => handleShortLongAnswer(question.id, question.isShortAnswer, e.target.checked)}
+                      isDisabled={question.isShortAnswer}
+                    >
+                      Long Answer
+                    </Checkbox>
+                  </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)